app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  has_secure_password
+  validates :username, presence: true, uniqueness: true, length: { minimum: 3 }
+  validates :password, presence: true, length: { minimum: 6 }
+end
```

Comment:
The code looks good and follows common practices for user authentication and validation in Ruby on Rails.